docs/devel/html.rst

```diff
@@ -93,7 +93,7 @@ selector to process all elements.
 .. note::
 
    Right now, only text of the elements is extracted. This add-on doesn't support localization
-   of element attributes or elements with childs.
+   of element attributes or elements with children.
```

Comment:
This does not make sense. A different word should be chosen, IMO. Something with sub- ?

Reply:
child elements is established in XML, so it looks okay to me.